src/retryable.ts

```diff
@@ -115,6 +121,8 @@ export default function retryable<Value = unknown>(action: Action<Value>): Promi
 
 		retry.setCount = resetRetryCount.bind(null, true);
 
+		retry.cancel = retryCancel;

```

Comment:
Also, please, add `cancel(): void` method to the `Retryer` type (see [`src/typings/retryer.ts`](https://github.com/parzh/retryable/blob/master/src/typings/retryer.ts))